src/app/features/profile/index.js

```
@@ -0,0 +1,18 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
```

ESLint

[error] 'createAsyncThunk' is defined but never used.

(no-unused-vars)